index.js

```diff
@@ -222,8 +222,10 @@ async function run() {
     if (waitForMinutes > MAX_WAIT_MINUTES) {
       waitForMinutes = MAX_WAIT_MINUTES;
     }
-    const forceNewDeployment = core.getInput('force-new-deployment', { required: false }) || false;
 
+    const forceNewDeployInput = core.getInput('force-new-deployment', { required: false })
+    const forceNewDeployment = forceNewDeployInput != undefined && forceNewDeployInput.toLocaleLowerCase === "true" ? true : false;
```

Comment:
let's use `.toLowerCase` instead, since we're not passing in a locale value

Reply:
also, nit: I'd prefer we wrap the 2 conditions of the ternary operator in parenthesis to make it clear they're being evaluated together to determine the return condition